Soccar [Game-Server]/src/nl/soccar/gameserver/controller/socnet/message/PlayerSyncMessage.java

```diff
@@ -0,0 +1,65 @@
+package nl.soccar.gameserver.controller.socnet.message;
+
+import nl.soccar.gameserver.controller.socnet.message.handler.PlayerSyncMessageHandler;
+import nl.soccar.socnet.message.Message;
+import nl.soccar.socnet.message.MessageConstants;
+import nl.soccar.socnet.message.MessageEvent;
+
+@MessageEvent(id = MessageConstants.SYNC_POSITION_PLAYER_MESSAGE_ID, handler = PlayerSyncMessageHandler.class)
+public final class PlayerSyncMessage extends Message {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)